lib/src/components/switch/switch.dart

```diff
@@ -6,6 +6,13 @@ import '../../robot/client.dart';
 /// Switch represents a set of finite positions.
 ///
 /// For more information, see [Switch component](https://docs.viam.com/dev/reference/apis/components/switch/).
+class PositionsInfo {
+  final int numberOfPositions;
+  final List<String>? labels;
+
+  PositionsInfo({required this.numberOfPositions, this.labels});
+}
+
```

Comment:
Because this class mirrors the attributes in `GetNumberOfPositionsResponse`, it's probably easier to typealias (see [camera example](https://github.com/viamrobotics/viam-flutter-sdk/blob/98ea2d6eb538d76eb137cdfbd677bb6cee135242/lib/src/components/camera/camera.dart#L9))